packages/core-js-sdk/src/httpClient/index.ts

```diff
@@ -36,6 +40,16 @@ const createDescopeHeaders = () => {
   };
 };
 
+const isJson = (value?: string) => {
+  try {
+    value = JSON.parse(value);
```

Comment:
What's the harm of adding it always?

Reply:
its usually bad practice on non POST/PUT and sometimes they may not send json body. i wouldn't want to change anything